src/getMetroConfig.js

```diff
@@ -8,49 +8,49 @@
 const fs = require('fs')
 const exec = require('child_process').execSync
 const dedent = require('dedent-js')
-const getDependencyPath = require('./getDependencyPath')
 
 const mapModule = name =>
     `'${name}': path.resolve(__dirname, 'node_modules/${name}')`
 
-const mapPath = path =>
-    `/${path.replace(
-        /\//g,
-        '[/\\\\]',
-    )}[/\\\\]node_modules[/\\\\]react-native[/\\\\].*/`
+const mapPath = path => `/${path.replace(/\//g, '[/\\\\]')}[/\\\\].*/`
 
-module.exports = symlinkedDependencies => {
-    const symlinkedDependenciesPaths = symlinkedDependencies.map(
-        getDependencyPath,
+const flatten = arr =>
+    arr.reduce(
+        (flatDependencies, dependencies) => [
+            ...flatDependencies,
+            ...dependencies,
+        ],
+        [],
     )
 
-    const peerDependenciesOfSymlinkedDependencies = symlinkedDependenciesPaths
+const unique = arr => Array.from(new Set(arr))
```

Comment:
We could move these to a lib file or pull in an external dependency to add the functionality.